day_9/dictionary_exercise_9_grading_program.py

```diff
@@ -0,0 +1,35 @@
+student_scores = {
+    'Harry': 88,
+    'Ron': 78,
+    'Hermione': 95,
+    'Draco': 75,
+    'Neville': 60
+}
+
+def score_to_grade(score):
+    if score in range(91, 101):
```

Comment:
This code work in next way: 

create an iterator that starts with 91. 
check if score is equal to 91, if not check generate next value (which is 92) and check with it and repeat. 

for example if range was 10000 numbers it will perform up to 10000 comparisons. 

It's much better just to 2 comparisons: 
```
if score >= 91 and score < 101:
```